### ISSUE TITLE
RIDE Log should get messages written to STDOUT and STDERR

### ISSUE THREAD

**Issue by yanne** [robotframework/RIDE#1179]
> _Originally submitted to [Google Code](http://code.google.com/p/robotframework-ride/issues/detail?id=1140) by @mkorpela on 28 Nov 2012_

Things to remember:
- --debugconsole 
- log_to_console option

**Comment by yanne** [comment]
> *Originally submitted to [Google Code](http://code.google.com/p/robotframework-ride/issues/detail?id=1140#c1) by @pekkaklarck on 17 Dec 2012*


Unfortunately no time to get this done for 1.0. The original description is a bit brief so the motivation behind this enhancement is explained below:

1) Currently errors can end up to the console (stdout/stderr) or into RIDE's log. It would be good to have all errors in one place and RIDE's log is definitely better place.

2) When starting RIDE using the desktop shortcut on Windows, errors written to stdout/stderr are not visible anywhere. If they were written into RIDE's log, debugging problems would be easier.

3) In general it would be better to have no extra outputs on console.

The original idea was to just intercept stdout and stderr unconditionally, but when we looked at this a little more we noticed that there are some cases, listed in the original description, when that is not a good idea. Need to still think should we intercept std streams automatically in other cases or should we perhaps make that configurable with a command line option. In the latter case the Windows shortcut should obviously enable intercepting automatically.
